pulp_container/tests/functional/api/test_remote_filter_pull_through.py

```diff
@@ -0,0 +1,63 @@
+import subprocess
+import pytest
+
+from pulp_container.tests.functional.constants import (
+    REGISTRY_V2,
+    PULP_HELLO_WORLD_REPO,
+    PULP_FIXTURE_1,
+)
+
+
+@pytest.fixture
+def pull_and_verify(
+    capfd,
+    delete_orphans_pre,
+    local_registry,
+    registry_client,
+):
+    def _pull_and_verify(images, pull_through_distribution, includes, excludes):
+        distr = pull_through_distribution(includes, excludes)
+        for image_path in images:
+            remote_image_path = f"{REGISTRY_V2}/{image_path}"
+            local_image_path = f"{distr.base_path}/{image_path}"
+
+            if excludes and "hello-world" not in image_path:
```

Comment:
What is `hello-world`? It came out of nowhere. Is not it better to use the following structure?
```
if image_path not in expected_paths:
   blabla
```